src/pages/EditList.jsx

```diff
@@ -0,0 +1,88 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+import { Pagination, styled, css, Box } from "@mui/material";
+
+import PokemonCard from "../components/PokemonCard";
+import { useFetchLocalApi } from "../hooks/useFetchLocalApi";
+
+const StyledBox = styled(Box)(
+  ({ theme }) =>
+    css`
+      height: 100%;
+      padding: 20px;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      background-color: ${theme.palette.background.contrast};
+    `
+);
+
+const StyledContent = styled(Box)(
+  css`
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+  `
+);
+
+const Heading = styled("h1")(
+  css`
+    color: black;
+  `
+);
+
+const EditList = () => {
+  const [pokedex, setPokedex] = useState([]);
+  const [url, setUrl] = useState(
+    `https://pokeapi.co/api/v2/pokemon/?offset=0&limit=15`
```

Comment:
Myślę że jedyne co musimy przechowywać w stanie to parametr i przekazać go do template string.